app/uk/gov/hmrc/economiccrimelevyregistration/controllers/NotableErrorController.scala

```diff
@@ -75,4 +76,13 @@ class NotableErrorController @Inject() (
   def assistantCannotRegister: Action[AnyContent] = authoriseAssistantsAllowed { implicit request =>
     Ok(assistantCannotRegisterView())
   }
+
+  def organisationAlreadyRegistered: Action[AnyContent] = authoriseWithoutEnrolmentCheck { implicit request =>
+    val eclRegistrationReference = request.eclRegistrationReference.getOrElse(
+      throw new IllegalStateException("ECL registration reference not found in request")
+    )
```

Comment:
Unfortunately this won't work - the ecl ref doesn't come from auth in this scenario, it comes from the get subscription status API